Dockerfile.explorer

```
@@ -2,14 +2,21 @@ FROM mcr.microsoft.com/dotnet/core/sdk:3.1 AS build-env
 WORKDIR /app
 
 # Copy csproj and restore as distinct layers
-COPY ./Libplanet.Explorer/Libplanet.Explorer.csproj ./Libplanet.Explorer/
+# Note that it's ordered by the least frequently changed
+COPY ./Libplanet.Stun/Libplanet.Stun.csproj ./Libplanet.Stun/
+RUN dotnet restore Libplanet.Stun
+COPY ./Libplanet.RocksDBStore/Libplanet.RocksDBStore.csproj ./Libplanet.RocksDBStore/
+RUN dotnet restore Libplanet.Stun
```

Is it intended restoring `Libplanet.Stun` one more time though it was already restored and it copied `Libplanet.RocksDBStore`?

Or did you intend this?

```suggestion
RUN dotnet restore Libplanet.RocksDBStore
```

---

It's mistake of course! Thanks!

---

@moreal Amended: 19bf389a7e608f00db559ed884615473492d520a.